ironfish/src/rpc/routes/node/getStatus.ts

```diff
@@ -18,6 +18,8 @@ export type GetStatusResponse = {
     status: 'started' | 'stopped' | 'error'
     version: string
     git: string
+    heapUsed: number
+    rss: number
   }
```

Comment:
I'm not 100% sure but I kind of want this to go into a memory section since this is for the process and not the node.

Reply:
That makes sense. I moved the numbers.